config/default/kustomization.yaml

```diff
@@ -159,8 +195,10 @@ patches:
 - path: manager_auth_proxy_patch.yaml
 - path: isvc_mutatingwebhook_cainjection_patch.yaml
 - path: isvc_validatingwebhook_cainjection_patch.yaml
-- path: trainedmodel_mutatingwebhook_cainjection_patch.yaml
+- path: inferencegraph_validatingwebhook_cainjection_patch.yaml
```

Comment:
inference graph validation webhook was not there before?

Reply:
It was there. It's just the file name is wrong.